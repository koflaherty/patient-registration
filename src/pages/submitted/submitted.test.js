import React from 'react';
import renderer from "react-test-renderer";
import Submitted from "./submitted";
import { MemoryRouter } from 'react-router-dom';

it('renders snapshot', () => {
    const tree = renderer
        .create(<MemoryRouter><Submitted/></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});