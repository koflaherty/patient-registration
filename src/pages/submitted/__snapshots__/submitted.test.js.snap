// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<div>
  <h1>
    Registration Complete
  </h1>
  <p>
    Someone will be with you shortly.
  </p>
  <a
    href="/"
    onClick={[Function]}
  >
    Register Another Patient
  </a>
</div>
`;
